CAIPs/caip-27.md

```diff
@@ -45,13 +47,14 @@ uppercase in this document are to be interpreted as described in [RFC
 
 ### Definition
 
-The JSON-RPC provider is able to invoke a single JSON-RPC request accompanied
-by a [CAIP-2][] compatible `chainId` scoped by the [sessionId][CAIP-171] of
-a pre-existing session.
+The JSON-RPC provider is able to invoke a single JSON-RPC request accompanied by
+a [CAIP-2][] compatible `chainId`, and optionally scoped by the
```

Comment:
🤔 can CAIP-27 target entire namespaces (as in all chainIds authorized under that namespace)? I wasn't aware of this... What are the use cases here? You want to simultaneously invoke a method across multiple chains? 

It doesn't seem like my changes introduce a need for more clarity on this point?

Reply:
tbh i am wondering if it even should anymore, or if `wallet:<namespace>` makes more sense as the target in every usecase that currently exists.  I'm currently working with a non-evm group and it seems like there, the `wallet:<namespace>` approach makes more sense.  

At the time we wrote this into CAIP-27 originally, I thought maybe in DAG, coordination-chain, or sharded VM architectures it might not always make sense to target a specific network, but I have yet to encounter a namespace that *actually* works this way (i.e. has namespace-wide execution), there always seem to be subnets or private networks that still need a distinct targeting/routing system, and whatever non-chain tangle/dag/DHT/thing is always just mainnet/chainId 1 in every context I've yet read documentation about...

Reply:
we should probably add non-normative guidance warning people NOT to use namespace-wide scopes unless they know what they're doing OR constrain it normatively to CAIP-2 scopes only, in a separate PR.